modules/phase_field/examples/rigidbodymotion/tests

```diff
@@ -3,27 +3,27 @@
     type = RunApp
     input = 'AC_CH_Multigrain.i'
     cli_args = '--check-input Mesh/uniform_refine=0 Executioner/Adaptivity/initial_adaptivity=0'
+
     # This example uses ShapeElementUserObject which contains a
-    # mooseWarning() that is treated as an error in dbg mode.
-    # Therefore only check this test in opt mode.
-    method = 'OPT'
+    # mooseWarning()
+    executable_pattern = 'phase_field-\w+$'
```

Comment:
There are two other tests in the same file that do the same thing, so it doesn't make sense to just change one.  These are just `--check-input` tests after all.  The rationale for running them only in opt mode is that if an input file syntax change breaks the test, it should break it in all modes, so we only need to run these tests in opt mode.  Also, I marked several of them as opt-only because even running them with `--check-input` was causing timeouts... so be careful if you undo any of those restrictions.